openpype/hosts/resolve/api/plugin.py

```diff
@@ -435,13 +438,12 @@ def load(self, files):
                 handle_start = version_data.get("handleStart", 0)
                 handle_end = version_data.get("handleEnd", 0)
                 frame_start_handle = frame_start - handle_start
-                frame_end_handle = frame_start + handle_end
+                frame_end_handle = frame_end + handle_end
                 database_frame_duration = int(
                     frame_end_handle - frame_start_handle + 1
                 )
-                if source_duration >= database_frame_duration:
-                    source_in += handle_start
-                    source_out -= handle_end
```

Comment:
I'm pretty sure this if statement had a particular reason @jakubjezek001 ?

The large comment above it I think explains it. We need the if statement for representations of a version that exclude the handles even though other representations of the version (and its version data) do have the handles. So we check: if the video is shorter than the database tells us - then we assume this particular representation has already excluded handles.

Technically it's an ingestion issue where the representation is lacking data about its contents - but this was the only sane workaround I believe?